packages/fxa-react/configs/tailwind.js

```diff
@@ -63,24 +49,9 @@ module.exports = {
         100: '25rem',
       },
       inset: {
-        '1/2': '50%',
-        '-px': '-1px',
-        '-3': '-0.75rem',
-        '-50': '-11.50rem',
-        '-52': '-13rem',
-        0.5: '0.125rem',
-        3: '0.75rem',
-        6: '1.25rem',
-        10: '2.5rem',
+        50: '11.50rem',
         55: '13.75rem',
         ten: '10%',
-        full: '100%',
-      },
-      top: {
-        2.5: '0.6rem',
-      },
-      left: {
-        3.5: '.87rem',
       },
```

Comment:
These custom configs for top and left were so close to the TW defaults that I felt it was safe to remove.
Here are the defaults:
`top-2.5: 0.625rem`
`left-3.5: 0.875rem`

Reply:
`top-2.5` is used in both the `AlertBar` component and `.arrow` class in `tailwind.css` in payments.
`left-3.5` is also used in the `.arrow` class in `tailwind.css` in payments.

Reply:
> `top-2.5` is used in both the AlertBar component and `.arrow` class in `tailwind.css` in payments.
`left-3.5` is also used in the `.arrow` class in `tailwind.css` in payments.

Does this make a difference visually? If we can use the default values that's definitely preferred.